src/components/watchers-form.vue

```diff
@@ -24,7 +24,7 @@
               aria-required="true"
             />
 
-            <form-multi-select
+            <!-- <form-multi-select
```

Comment:
you can delete this commented snippet of code...

Reply:
Good catch, removed